cmd/kops/create_cluster.go

```diff
@@ -475,7 +475,7 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 
 		// Default to a DNS name for the bastion
 		cluster.Spec.Topology.Bastion = &api.BastionSpec{
-			BastionPublicName: "bastion-" + clusterName,
+			BastionPublicName: "bastion." + c.DNSZone,
```

Comment:
I think this was correct with `"bastion." + clusterName` and the dot.  I'm sorry I didn't get to reviewing this sooner - I have reopened #1322 and commented here: https://github.com/kubernetes/kops/issues/1322#issuecomment-271294740 

Because we're trying to release kops 1.5 and this is a blocker IMO, I opened up #1404 as well.  I wouldn't normally do that (sorry), but maybe see if that solves the problem?